.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,19 @@
+name: Run Tests
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.11'
+      - name: Set up environment
+        run: |
+          curl -sSL https://install.python-poetry.org | python3 -
```

Comment:
You can replace this line by `make build` I guess it would work because this command just install poetry but it doesn't setup the env or you can add `poetry install --without evaluation`